README.md

```diff
@@ -88,8 +88,32 @@ The following steps are intended to help get a new user up and running the JPO C
 
 ### Debugging
 
-Note that it is recommended to work with the Python API from a [virtual environment](https://docs.python.org/3/library/venv.html). See [Visual Studio Code](https://code.visualstudio.com/docs/python/environments) documentation for more information on how to set up a virtual environment in VS Code.
+Note that it is recommended to work with the Python API from a [virtual environment](https://docs.python.org/3/library/venv.html). 
 
+#### Setting up a virtual environment from the command line
+1. Verify that you have Python 3.12.2 installed on your machine by running `python3 --version`. If not, download and install it from the [Python website](https://www.python.org/downloads/).
+2. Open a terminal and navigate to the root of the project.
+3. Run the following command to create a virtual environment in the project root:
+
+    ```bash
+    python3 -m venv .venv

```

Comment:
Should this be updated to be 'python3.12 -m venv .venv'? My python3 version was outdated and so this command created a virtual environment with python3.11 instead of 3.12. Alternatively we could include the instruction to update which version the python3 command uses.

Reply:
The 'python3' command is not valid on Windows ([source](https://stackoverflow.com/questions/48355319/cmd-windows-python-command-works-but-python3-doesnt-although-my-python-ver)), so I have modified this to include both Linux & Windows commands.

We may want to consider adding a dev container to the project that uses Python 3.12, but this may be out of the scope of this PR.